### DISCUSSION TITLE
[FR] Allow `craft sections/create` to run non-interactively via options

### DISCUSSION THREAD

**Discussion by kylecotter** [craftcms/cms#13864] [Ideas] [answered]
Would be nice if we could run the `craft sections/create` command and pass some options for name/handle/type/anything else required so we can run it non-interactively.

Use case is our internally content modeling tool can just spit out a bunch of these commands for a dev to run after a content model has been approved for buildout.

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 4.6! (0b853687d26ddee8094e4039f75539b32d42b3b0)

**Reply by brandonkelly** [reply]
Craft 4.6.0 is out now with that change.
